Automattic/jetpack
# My Jetpack: Replace WordPress.com logo from PNG to SVG

Fixes #24500

#### Changes proposed in this Pull Request:

* Replace `wordpress.png` to `wordpress.svg`.
* Set `max-width` for `wordpress.com` logo.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `My Jetpack`
* Navigate to `Connection` page trough footer.
* You should not see the `WordPress.com` logo blurred.

#### DEMO

![image](https://user-images.githubusercontent.com/1663717/174670336-5b1932a2-6030-4c23-85a8-78310b4d9615.png)
